addons/lamg/CfgWeapons.hpp

```diff
@@ -0,0 +1,20 @@
+class CfgWeapons {
+    class LMG_Mk200_F;
+    class CLASS(LMG_LAMG_F): LMG_Mk200_F {
```

Comment:
```suggestion
    class CLASS(LMG_LAMG): LMG_Mk200_F {
```
`F` is Futura, so we don't need it.